editor/gui/editor_file_dialog.cpp

```diff
@@ -1528,7 +1528,13 @@ void EditorFileDialog::_recent_selected(int p_idx) {
 }
 
 void EditorFileDialog::_go_up() {
-	dir_access->change_dir(get_current_dir().get_base_dir());
+	String cur_dir = get_current_dir();
+
+	if (cur_dir.ends_with("/")) {
+		cur_dir = cur_dir.substr(0, cur_dir.length() - 1);
```

Comment:
```suggestion
		cur_dir = cur_dir.left(-1);
```
Has the same effect

Reply:
thank you for the advice, now `left(-1)`